woocommerce/wc-api-dev
# Overwrite order URL for admins in order details emails

See https://github.com/Automattic/wp-calypso/issues/16660.

The emails being sent to the store admin (which is also our only notification right now) links to `wp-admin` instead of Calypso. This PR adds a hotfix for overwriting the templates with our own, linking to WP.com.

To Test:
* Upload/install this branch over your existing `wc-api-dev` plugin.
* Go through the check-out process and make sure you receive an admin order notification email.
* Make sure that the link on the Order # goes to WP.com (make sure you are looking at the admin notification, and not the notification that is sent to the customer).
* To test the plaintext email (which is not currently an option we offer on .com, but I provided here anyway), you can go to `wp-admin > WooCommerce > Settings > Emails > New Order` and then set it to plaintext.
* Repeat the above steps.
